src/Symfony/Component/DependencyInjection/Exception/ContextElementNotFoundException.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Exception;
+
+/**
+ * @author Maxime Steinhausser <maxime.steinhausser@gmail.com>
+ */
+class ContextElementNotFoundException extends InvalidArgumentException
```

Comment:
Dont get me wrong ;-) but consider extending `RuntimeException`. `$key` is a valid argument by [design](https://github.com/symfony/symfony/pull/19646/files#diff-b6d4082e39c324aadd2aaf7e4a36511fR64).